src/javascript/app/pages/user/accounts.js

```diff
@@ -86,27 +86,44 @@ const Accounts = (() => {
     const getCompanyCountry = account => Client.getLandingCompanyValue(account, landing_company, 'country');
 
     const populateNewAccounts = (upgrade_info) => {
+        // TODO Populate multiple account opening.
```

Comment:
what is this comment?

Reply:
Ah :D I forgot to remove it after I was done with the task.